salt/modules/boto_vpc.py

```diff
@@ -837,10 +837,10 @@ def subnet_exists(subnet_id=None, name=None, subnet_name=None, cidr=None,
         subnets = conn.get_all_subnets(**filter_parameters)
         log.debug('The filters criteria {0} matched the following subnets:{1}'.format(filter_parameters, subnets))
         if subnets:
-            log.info('Subnet {0} exists.'.format(subnet_id))
+            log.info('Subnet {0} exists.'.format(subnet_name or subnet_id))
```

Comment:
This will produce `Subnet True exists`.  Did you mean `subnet_name if subnet_name else subnet_id`?